src/Controllers/AddProductController.php

```diff
@@ -23,6 +24,82 @@ public function __construct($productModel)
 
     public function __invoke(Request $request, Response $response, array $args)
     {
-        return $response;
+        $newProductData = $request->getParsedBody();
```

Comment:
Yeah we didn't discuss this or plan it but i think it makes more sense that the product will be sent like this  
`{
    "product": 
        {
            "sku": "abcdef123456",
            "name": "test_name_1",
            "price": "99.99",
            "stockLevel": "1"
        }
}`
In the body. $newProductData['product'] will result in the same as it worked previously.